paper.tex

```diff
@@ -81,9 +81,53 @@ \section{Method}
 
-\section{Results}
+\section{Results + Discussion}
+In the following we will have a look at the results of the aforementioned experiment.
 
-\section{Discussion}
+
+In fig. \ref{stable_diffusion} can be seen how the different processors and their combinations effect an image generated by the stable diffusion generator. The bars are shown in packs of three where the most left one represents the nahrawy evaluator, the middle one is resnet18 and lastly on the right is the umm-maybe evaluators respectively. The different colors represent different output. Black simbolizes that the image before processing was detected as AI-generated and after processing as well. Grey shows that both are detected to be human made, whereas for red the process did the exact opposite of what it should do and fool the detector in a way that after processing it is seen as AI-generated whereas before as human made. Lastly the green bar does exactly what is desired. \\
+It can be seen in fig. \ref{fig:stable_diffusion}, that only the nahrawy evaluator has a little bit of red in it but also it is the only one with the output of 'human-human'. Compared to the dall-E generator in fig. \ref{fig:dallE} the amount of green is relatively low. There are also other differences as here only umm-maybe achieved 'human-ai' but also 'human-human'. 
+It is visible that the gaussian noise image processor as well as the salt-and-pepper one are the best processors. This can be seen as they alone achieve a great result as well as their combinations with other ones. It is also worth mentioning that these result are only achieved when using the nahrawy evaluator. Whereas resnet18 which can be seen in both figures has only the output of 'ai-ai'.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=\linewidth]{images/stable-diffusion.png}
+	\caption{Stable Diffusion with different processors and nahrawy, resnet18 and umm-maybe as evaluators}
+	\label{fig:stable_diffusion}
+\end{figure}
+
+\begin{figure}
+	\centering
+	\includegraphics[width=\linewidth]{images/dalle.png}
+	\caption{Dall-E with different processors and nahrawy, resnet18 and umm-maybe as evaluators}
+	\label{fig:dallE}
+\end{figure}
+
+
+In fig. \ref{fig:gpt2} the text processors can be seen. The colors stay the same and the concept as well, where the left bar is using the radar evaluator and the right one the roberta base one. It is visible that the double-whitespace processor alone has little effect whereas in combination with the typo one the result rise enourmously. This is due to the typo processor, since it alone also achieve great results. Here is also one evaluator, which produces only a black bar. 
```

Comment:
```suggestion
In fig. \ref{fig:gpt2} the text processors are plotted. The colors stay the same and the concept as well, where the left bar is using the radar evaluator and the right one the roberta based one. It is visible that the double-whitespace processor solely has little effect whereas in combination with the typo processor the result rises enormously. This is due to the typo processor since it alone also achieves great results. The radar evaluatoronly produces only black bars. 
```

Reply:
Is 'based' here right ?